source/ranges.tex

```diff
@@ -1151,18 +1151,18 @@
     requires (K == subrange_kind::sized || !SizedSentinel<S, I>)
   class subrange : public view_interface<subrange<I, S, K>> {
   private:
-    static constexpr bool StoreSize =                   // \expos
+    static constexpr bool StoreSize =                      // \expos
       K == subrange_kind::sized && !SizedSentinel<S, I>;
-    I begin_ = I();                                     // \expos
-    S end_ = S();                                       // \expos
-    iter_difference_t<I> size_ = 0;                     // \expos; present only
-                                                        // when \tcode{StoreSize} is \tcode{true}
+    I begin_ = I();                                        // \expos
+    S end_ = S();                                          // \expos
+    @\placeholder{make-unsigned-like-t}@(iter_difference_t<I>) size_ = 0;  // \expos; present only
```

Comment:
\placeholdernc ?

Reply:
... and what should be used in this case?

Reply:
changed also.